Examples/Examples/Cart/CartViewController.m

```diff
@@ -159,8 +158,16 @@ - (IBAction)checkoutPressed:(id)sender {
         return;
     }
 
-    NSString *customerId = [[NSUserDefaults standardUserDefaults] stringForKey:kCachedCustomerID];
-    [self createPaymentIntentWithCustomerId:customerId];
+    [self.activityIndicator startAnimating];
+    [[APIClient sharedClient] createAuthenticationTokenWithCompletionHandler:^(NSError * _Nullable error) {
+        if (error) {
+            [self showAlert:NSLocalizedString(@"Fail to request token.", nil) withTitle:nil];
```

Comment:
Shall we also utilise the error from the completion block for the alert message?